### DISCUSSION TITLE
PCB order

### DISCUSSION THREAD

**Discussion by Mikko222** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#537] [General]
Hi.. i hade to write it here since i couldnt comment on the OSHWLab site. Havent used it enough to be able to comment there.
Anyways... it seems like i there are som parts out of stock to be bale to order the PCB board.. Does anyone know if this happend from time to time and if so for how long does it take before it will be restocked again? 

Gonna give this project a try but have to wait for the parts =)

![image](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/135755693/0d02dec3-1e01-4001-9531-e1bb4c52cb2d)

**Comment by visualapproach** [comment]
I'd order just the PCB and source the other parts from somewhere else. If you even need them.

**Reply by Mikko222** [reply]
hmm ok which one is the pcb board? the first of the two available? (first time im useing OSHWLab sadly so bear with me :/)

**Reply by Mikko222** [reply]
@visualapproach sorry to bother you again. But do you know where we can order just the pcb board?

**Reply by visualapproach** [reply]
Download PCB files from here: [https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/tree/master/Code/PCB](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/tree/master/Code/PCB)
Upload and order here or elsewhere [https://jlcpcb.com/](https://jlcpcb.com/)

**Reply by Mikko222** [reply]
Thanks for your help 🙏. Now i managed to order it without any problem =)

**Comment by eden919191** [comment]
Hi,

Don't know if someone have spare pcb board ? 

Thanks!

**Reply by challo2018** [reply]
I have some PCBs left. Where to ship?

**Comment by KipiKland** [comment]
Hi i have 3 pcb V2 available, i can send them from Switzerland virtually anywhere 
![IMG_6103](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/63462251/7e6de400-ba98-4ceb-bad3-c7ea8ea291e5)

**Reply by JohnyJohn73** [reply]
interested, living in france, how can I contact you ? (also french speaking)

**Reply by KipiKland** [reply]
Pas de problème, mon mail : k i p i k at  k  i p i k  point com

**Reply by Richard7619** [reply]
KipiKland Sent you an email also. 👍

**Reply by KipiKland** [reply]
@JohnyJohn73 , @Richard7619 , sent you a mail.

**Reply by dklemm** [reply]
@KipiKland I just dropped you an email.  to see if you have another spare you can ship to the UK.  🙇‍♂️
